.circleci/config.yml

```diff
@@ -721,16 +721,22 @@ jobs:
       - checkout
       - run:
           name: Install Go linters
-          command: make scripts/go/bin/golangci-lint scripts/go/bin/revive scripts/go/bin/gosec
+          command: |
+            pushd /tmp
+            curl -fLO https://github.com/golangci/golangci-lint/releases/download/v1.24.0/golangci-lint-1.24.0-linux-amd64.tar.gz
```

Comment:
Sounds reasonable. But once this is working we need to figure out how CI runs the same version as devs to locally.

Reply:
@bergquist Sure. The linting time has been reduced to ~3 minutes in CI, so I suspect using the official build is what helped.